services/matrix-admin/src/styles.css

```
@@ -0,0 +1,610 @@
+  --color-danger: #f97066;
+  --color-danger-dim: rgba(249, 112, 102, 0.12);
+  --color-danger-border: rgba(249, 112, 102, 0.2);
+  --color-warning: #f59e0b;
+  --color-warning-dim: rgba(245, 158, 11, 0.12);
+  --color-warning-border: rgba(245, 158, 11, 0.25);
+  --color-info: #3b82f6;
+  --color-info-dim: rgba(59, 130, 246, 0.12);
+  --color-accent-purple: #667eea;
+  --color-accent-purple-dim: rgba(102, 126, 234, 0.14);
+
+  --font-sans: "Inter", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)